### DISCUSSION TITLE
New features

### DISCUSSION THREAD

**Discussion by MMC227** [Joaopeuko/Mql5-Python-Integration#5] [Ideas]
Ola João,

Gostaria de continuar o desenvolvimento deste projeto?

**Comment by Joaopeuko** [comment]
Olá MMC227, tudo bem?
Obrigado pela mensagem.

Eu tenho grande interesse em continuar o projeto, e sempre que possível trabalho nele.
Se você tiver sugestão ou interesse em adicionar algo, me envie uma mensagem.

Muito obrigado,
Joao.
